src/bp-core/admin/js/help.js

```diff
@@ -10,18 +10,29 @@
 		'load',
 		function() {
 
+			bbapp_help_empty_everything();
+			var article_id = getArticleParam();
+
+			// Hide Both Section by Default.
+			$('#bp-help-main-menu-wrap').hide();
+			$('.bp-help-content-wrap').hide();
+
 			var bp_help_wpapi = new WPAPI({ endpoint: 'https://buddyboss.com/resources/wp-json' });
-			//var bp_help_wpapi = new WPAPI({ endpoint: 'http://localhost/buddyboss/wp-json' });
+			
 			bp_help_wpapi.docs = bp_help_wpapi.registerRoute( 'wp/v2', '/docs/(?P<id>)', {
-				params: [ 'before', 'after', 'author', 'parent', 'post', 'order', 'orderby' ]
+				params: [ 'before', 'after', 'author', 'parent', 'post', 'order', 'orderby', 'per_page' ]
 			} );
 			var bp_help_page_url = new URL(window.location.href);
 
-			if ( $( '#bp-help-main-menu-wrap' ).length ) {
+			/**
+			 * Loads Main Page.
+			 */
+			if (!article_id) {
 
 				$( '#bp-help-main-menu-wrap' ).addClass( 'loading' ).html('<div class="content-loader"><div></div><div></div><div></div></div>');
+				$('#bp-help-main-menu-wrap').show();
 
-				bp_help_wpapi.docs().parent(0).order('asc').orderby('menu_order').then(function (docs) {
+				bp_help_wpapi.docs().parent(0).order('asc').orderby('menu_order').per_page(100).then(function (docs) {
```

Comment:
@sourabhmatolia If you added a pagination here then how I get the `101` record. I don't see the pagination code here.

Reply:
Previously we were showing 20 main categories.
Now we have changed that to show max 100 categories.
We don't have a requirement here to show pagination because we don't expect more than 20-30 Categories here.
In the future, if needed we can tweak this if needed. 

For now, I think this is good. Thanks.